pyqtgraph/graphicsItems/PlotDataItem.py

```diff
@@ -530,7 +530,7 @@ def getData(self):
             if self.opts['logMode'][0]:
                 x = np.log10(x)
             if self.opts['logMode'][1]:
-                y = np.log10(y)
+                y = np.log10(1.0+y)
             #if any(self.opts['logMode']):  ## re-check for NANs after log
                 #nanMask = np.isinf(x) | np.isinf(y) | np.isnan(x) | np.isnan(y)
                 #if any(nanMask):
```

Comment:
I think it was removed because it is computationally expensive and unnecessary--Qt should simply ignore any points that are NaN. If you just want to avoid the warning message, then the calls to `np.log10()` can be wrapped with `warnings.catch_warnings`. Issue #144 will require a more complicated solution because InfiniteLine and LinearRegionItem are not log-aware.